pkg/sdk/testint/tables_integration_test.go

```diff
@@ -1034,14 +1052,39 @@ func TestInt_TablesShowByID(t *testing.T) {
 		require.Equal(t, id2, e2.ID())
 	})
 
-	t.Run("show by id: check fields", func(t *testing.T) {
+	t.Run("show by id: check schema evolution record", func(t *testing.T) {
```

Comment:
this test checks the new field in information_schema (that differs in name that the one added to `TableColumnDetails` -> result of `DescribeColumns`). Can we check this also? (even in the same test)